doc/source/orbit.rst

```diff
@@ -346,6 +346,52 @@ behavior
 
 .. image:: images/lp-orbit-integration-EzJz.png
 
+.. _fastchar:
+
+Fast orbit characterization
+-----------------------
+
+It is also possible to use galpy for the fast estimation of orbit parameters via the Staeckel 
+approximation (originally used by `Binney (2012) <http://adsabs.harvard.edu/abs/2012MNRAS.426.1324B>`_
+for the appoximation of actions in axisymmetric potentials), without performing any orbit integration. 
+The method uses the geometry of the orbit tori to estimate the orbit parameters. After initialising 
+an ``Orbit`` instance, this is done (as in the previous section) specifying ``analytic=True`` and 
+selecting ``type='staeckel'`` (default in vX.X of galpy).
+
+>>> o.e(analytic=True, type='staeckel')
+
+if running the above without integrating the orbit, the potential should also be specified
+in the usual way
+
+>>> o.e(analytic=True, type='staeckel', pot=mp)
+
+again, where ``mp`` is the Miyamoto-Nagai potential of :ref:`Introduction:
+Rotation curves <rotcurves>`. This interface automatically computes the necessary Delta 
+parameter based on the initial condition of the ``Orbit`` object.
+
+While this is useful and fast for individual ``Orbit`` objects, it is likely that users will
+want to rapidly evaluate the orbit parameters of large numbers of objects. It is possible
+to perform the orbital parameter estimation above through the :ref:`actionAngle <actionangle>` 
+interface. To do this, we need arrays of the phase-space points ``R``, ``vR``, ``vT``, ``z``, ``vz``, and 
+``phi`` for the objects. We can then optionally estimate the individual Delta parameter 
+for these phase-space points using
+
+>>> from galpy.actionAngle import estimateDeltaStaeckel
+>>> deltas = estimateDeltaStaeckel(mp, R, z, no_median=True)
+
+where ``no_median=True`` specifies that the function return the delta parameter at each given point
+rather than the median of the calculated deltas. The orbit parameters are then calculated by first 
+specifying an ``actionAngle`` instance (with an arbitrary delta parameter), then using the 
+``EccZmaxRperiRap`` method with the data points and the estimated delta array:
+
+>>> aAS = actionAngleStaeckel(pot=mp, delta=0.4)
+>>> e, Zmax, rperi, rap = aAS.EccZmaxRperiRap(R, vR, vT, z, vz, phi, delta=deltas)
+
+Here, delta can alternatively be either a single scalar value for all points, negating the need for
+the estimation at each point. This method is also applicable in the ``actionAngleIsochrone``, 
+``actionAngleSpherical``, and ``actionAngleAdiabatic`` modules. This method returns parameters at
+speeds as fast as 3 microseconds per object. 
+
```

Comment:
Is it really 3 microseconds? You could put an example with ``timeit`` to explicitly show this (so people can easily test this themselves)

Can we also add an example of using the actionAngleStaeckelGrid approach and how fast that is?